Doc/library/random.rst

```diff
@@ -177,6 +178,9 @@ Functions for sequences
 
    .. versionadded:: 3.6
 
+   .. versionchanged:: 3.9
+      Raises a :exc:`ValueError` if the total weights aren't more than zero.
```

Comment:
Similar to above.  Behavior was always undefined if any weight was negative.  There's no need to constrain other implementations to raise `ValueError` if their sum is `<= 0`, but there is a need to constrain them to raise `ValueError` if all weights are 0, because that's the _intended_ part.  So:

"Raises a :exc:`ValueError` if all weights are zero."

instead.